packages/ozone/src/mod-service/index.ts

```diff
@@ -1098,12 +1104,22 @@ export class ModerationService {
       )
     }
 
+    if (minPriorityScore != null && minPriorityScore > 0) {
+      builder = builder.where(
+        'moderation_subject_status.priorityScore',
+        '>=',
+        minPriorityScore,
+      )
+    }
```

Comment:
Are you sure we don't want to allow `priorityScore` to be `0`?

This would allow to distinguish "items with a priority score defined" from items "without a priority score (priority unknown)"

Reply:
yeah good point, I don't see a use-case for this right now but doesn't hurt to allow it.